app/views/presenters/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1> Presenter Dashboard </h1>
\ No newline at end of file
+<h1> Presenter Dashboard </h1>
```

Comment:
The presenter navigation has already been abstracted to partial '_presenter_nav.html.erb'